### ISSUE TITLE
driver.sh should provide less verbose output by default 

### ISSUE THREAD

**Issue by tpimh** [ClangBuiltLinux/continuous-integration#217]
Yeah, `set -x` is cool, but maybe it should be hidden behind an extra command line option (like `--debug`). As `driver.sh` gets bigger and more complicated, more output is produced and in most cases it's unnecessary unless you are debugging `driver.sh` itself.
I suggest only producing important information by default: versions of all utilities used, kernel version being built and a list of patches applied. Anything to add to this list?

**Comment by nickdesaulniers** [comment]
I agree that we can probably hide more things, but some parts are important to have in the output to debug when something goes wrong.  If you have specific sections that you think could be silenced, please call them out and let's discuss if `set +x` could be applied to those sections.
